---
title: "PC_pathway"
output: pdf_document
---


Load a bunch of packages a setup knitcitations
```{r}
library(ggplot2)
library(tidyr)
library(dplyr)
library(magrittr)
library(readr)
library(knitcitations)
cleanbib()
options("citation_format" = "pandoc")
library(cowplot)
```

load dataset 1 from citep("doi:10.3835/plantgenome2015.04.0025") and list of PC pathway genes

```{r, echo=FALSE}
tissue_exp <- read.delim("data/Dataset_S1.txt", stringsAsFactors=FALSE) 
pc_pathway <- read_csv("data/pc_pathway_genes.csv") # it can be dowloaded from Plant Metabolic Network

# subset list of PC pathway genes

pc_pathway_exp <- tissue_exp %>%
  inner_join(pc_pathway) %>% # and now we transform the dataset 
  gather(tissue, expression_value, -Maize_AGPv2_gene, -chromosome, -position_left, -position_right) # we could generate more grouping variables like root types etc.

```

Data is now ready to start plotting.

```{r}
pc_pathway_exp %>%
  filter(Maize_AGPv2_gene == "GRMZM2G170400") %>% # GO XPT1B!!
  filter(expression_value > 250) %>% # to reduce the number of tissues plotted
  ggplot(aes(x = reorder(tissue,expression_value), y = expression_value)) +
  geom_point() +
  coord_flip() 

```


lipids <- read_csv("data/lipid.csv") # it can be dowloaded from Plant Metabolic Network

# subset list of PC pathway genes

lipid_exp <- tissue_exp %>%
  inner_join(lipids) %>% # and now we transform the dataset 
  gather(tissue, expression_value, -gene, -chromosome, -position_left, -position_right) # we could generate more grouping variables like root types etc.

```

Data is now ready to start plotting.

```{r}
pc_pathway_exp %>%
  filter(Maize_AGPv2_gene == "GRMZM2G170400") %>% # GO XPT1B!!
  filter(expression_value > 250) %>% # to reduce the number of tissues plotted
  ggplot(aes(x = reorder(tissue,expression_value), y = expression_value)) +
  geom_point() +
  coord_flip() 

```


